packages/xstate-graph/src/graph.ts

```diff
@@ -13,8 +13,11 @@ import {
   StatePaths,
   AdjacencyMap,
   Segments,
-  ValueAdjMapOptions
+  ValueAdjMapOptions,
+  DirectedGraphEdge,
+  DirectedGraphNode
 } from './types';
+import { getChildren } from 'xstate/lib/stateUtils';
```

Comment:
What's the exact purpose of `@xstate/graph`? Seems like such an algorithm (`getChildren`) would, maybe, be a good candidate to be included here?

Reply:
Good call